.travis.yml

```diff
@@ -1,15 +1,17 @@
 language: php
 
+dist: trusty
+
 env:
   global:
     - COMPOSER_ROOT_VERSION="4.3.x-dev"
 
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL RECIPE_VERSION=1.0.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=^1.0.0 PHPCS_TEST=1 PHPUNIT_TEST=1
```

Comment:
Should we rather use the tilde in here `~` since caret would install 1.1

https://semver.mwl.be/#!?package=silverstripe%2Frecipe-cms&version=%5E1.0.0&minimum-stability=stable

Reply:
Good point. Just pushed a fix.